2005-02-21  Andrew Cagney  <cagney@gnu.org>

	Backported from mainline.
	* config/mips/linux.mt: Add symfile-mem.o.
	* config/sh/linux.mt, config/arm/linux.mt: Ditto.
	* config/m32r/linux.mt, config/m68k/linux.mt: Also.
	* config/powerpc/linux.mt, config/sparc/linux.mt: Likewize.
	* config/pa/linux.mt, config/ia64/linux.mt: Finally.
	* config/i386/linux64.mt, config/sparc/linux64.mt: But don't forget.

--- ./gdb/config/mips/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/mips/linux.mt	2005-02-21 11:37:44.000000000 -0500
@@ -1,6 +1,6 @@
 # Target: Linux/MIPS
 TDEPFILES= mips-tdep.o mips-linux-tdep.o corelow.o \
-	solib.o solib-svr4.o
+	solib.o solib-svr4.o symfile-mem.o
 DEPRECATED_TM_FILE= tm-linux.h
 
 SIM_OBS = remote-sim.o
--- ./gdb/config/sh/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/sh/linux.mt	2005-02-21 11:37:59.000000000 -0500
@@ -1,5 +1,5 @@
 # Target: Renesas Super-H running GNU/Linux
-TDEPFILES= sh-tdep.o monitor.o sh3-rom.o remote-e7000.o ser-e7kpc.o dsrec.o solib.o solib-svr4.o solib-legacy.o
+TDEPFILES= sh-tdep.o monitor.o sh3-rom.o remote-e7000.o ser-e7kpc.o dsrec.o solib.o solib-svr4.o solib-legacy.o symfile-mem.o
 DEPRECATED_TM_FILE= tm-linux.h
 
 SIM_OBS = remote-sim.o
--- ./gdb/config/arm/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/arm/linux.mt	2005-02-21 11:37:10.000000000 -0500
@@ -1,3 +1,3 @@
 # Target: ARM based machine running GNU/Linux
 DEPRECATED_TM_FILE= tm-linux.h
-TDEPFILES= arm-tdep.o arm-linux-tdep.o glibc-tdep.o solib.o solib-svr4.o solib-legacy.o
+TDEPFILES= arm-tdep.o arm-linux-tdep.o glibc-tdep.o solib.o solib-svr4.o solib-legacy.o symfile-mem.o
--- ./gdb/config/m32r/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/m32r/linux.mt	2005-02-21 11:37:32.000000000 -0500
@@ -1,5 +1,5 @@
 # Target: Renesas M32R running GNU/Linux
-TDEPFILES= m32r-tdep.o m32r-linux-tdep.o remote-m32r-sdi.o glibc-tdep.o solib.o solib-svr4.o solib-legacy.o
+TDEPFILES= m32r-tdep.o m32r-linux-tdep.o remote-m32r-sdi.o glibc-tdep.o solib.o solib-svr4.o solib-legacy.o symfile-mem.o
 DEPRECATED_TM_FILE= config/tm-linux.h
 
 SIM_OBS = remote-sim.o
--- ./gdb/config/m68k/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/m68k/linux.mt	2005-02-21 11:37:38.000000000 -0500
@@ -1,3 +1,3 @@
 # Target: Motorola m68k with a.out and ELF
-TDEPFILES= m68k-tdep.o m68klinux-tdep.o solib.o solib-svr4.o solib-legacy.o
+TDEPFILES= m68k-tdep.o m68klinux-tdep.o solib.o solib-svr4.o solib-legacy.o symfile-mem.o
 DEPRECATED_TM_FILE= solib.h
--- ./gdb/config/powerpc/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/powerpc/linux.mt	2005-02-21 11:37:53.000000000 -0500
@@ -1,6 +1,6 @@
 # Target: Motorola PPC on Linux
 TDEPFILES= rs6000-tdep.o ppc-linux-tdep.o ppc-sysv-tdep.o solib.o \
-	solib-svr4.o solib-legacy.o corelow.o
+	solib-svr4.o solib-legacy.o corelow.o symfile-mem.o
 DEPRECATED_TM_FILE= tm-linux.h
 
 SIM_OBS = remote-sim.o
--- ./gdb/config/sparc/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/sparc/linux.mt	2005-02-21 11:38:03.000000000 -0500
@@ -1,4 +1,4 @@
 # Target: GNU/Linux SPARC
 TDEPFILES= sparc-tdep.o sparc-sol2-tdep.o sparc-linux-tdep.o \
-	solib.o solib-svr4.o
+	solib.o solib-svr4.o symfile-mem.o
 DEPRECATED_TM_FILE= solib.h
--- ./gdb/config/sparc/linux64.mt.dist	2005-02-21 11:51:36.929467728 -0500
+++ ./gdb/config/sparc/linux64.mt	2005-02-21 11:51:46.784969464 -0500
@@ -1,5 +1,5 @@
 # Target: GNU/Linux UltraSPARC
 TDEPFILES= sparc64-tdep.o sparc64-sol2-tdep.o sparc64-linux-tdep.o \
 	sparc-tdep.o sparc-sol2-tdep.o sparc-linux-tdep.o \
-	solib.o solib-svr4.o
+	solib.o solib-svr4.o symfile-mem.o
 DEPRECATED_TM_FILE= solib.h
--- ./gdb/config/pa/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/pa/linux.mt	2005-02-21 11:37:49.000000000 -0500
@@ -1,3 +1,3 @@
 # Target: HP PA-RISC running Linux
-TDEPFILES= hppa-tdep.o hppa-linux-tdep.o glibc-tdep.o solib.o solib-svr4.o
+TDEPFILES= hppa-tdep.o hppa-linux-tdep.o glibc-tdep.o solib.o solib-svr4.o symfile-mem.o
 DEPRECATED_TM_FILE=tm-linux.h
--- ./gdb/config/ia64/linux.mt.dist	2005-02-21 11:35:45.000000000 -0500
+++ ./gdb/config/ia64/linux.mt	2005-02-21 11:37:24.000000000 -0500
@@ -1,4 +1,4 @@
 # Target: Intel IA-64 running GNU/Linux
 TDEPFILES= ia64-tdep.o ia64-aix-tdep.o ia64-linux-tdep.o \
-	solib.o solib-svr4.o solib-legacy.o
+	solib.o solib-svr4.o solib-legacy.o symfile-mem.o
 DEPRECATED_TM_FILE= tm-linux.h
--- ./gdb/config/i386/linux64.mt.dist	2005-02-21 11:50:21.020007720 -0500
+++ ./gdb/config/i386/linux64.mt	2005-02-21 11:50:32.116320824 -0500
@@ -1,5 +1,5 @@
 # Target: GNU/Linux x86-64
 TDEPFILES= amd64-tdep.o amd64-linux-tdep.o \
 	i386-tdep.o i387-tdep.o i386-linux-tdep.o glibc-tdep.o \
-	solib.o solib-svr4.o corelow.o
+	solib.o solib-svr4.o corelow.o symfile-mem.o
 DEPRECATED_TM_FILE= solib.h
